jwst/tweakreg/tweakreg_step.py

```diff
@@ -1,8 +1,4 @@
-"""
-JWST pipeline step for image alignment.
-
-:Authors: Mihai Cara
```

Comment:
Actual authors are captured in `git log` and I don't think this field is consistently used throughout package doc, so might as well remove for consistency. Hopefully not controversial.